src/rsmq/cmd/pop_message.py

```diff
@@ -29,4 +29,4 @@ def exec_command(self):
         if not result:
             raise NoMessageInQueue(self.get_qname)
         [message_id, message, rc, ts] = result
-        return {'id': message_id, 'message': message, 'rc': rc, 'ts': ts}
+        return {'id': message_id, 'message': message, 'rc': rc, 'ts': int(ts)}
```

Comment:
is this safe to do?

Reply:
`ts` represents a timestamp in millis, so it is always safe to covert it to integer

Reply:
just want to make sure we do not crash if this is somehow is not set or is set to a string